core/src/test/java/feign/FeignTest.java

```diff
@@ -218,6 +218,18 @@ public void customExpander() throws Exception {
   }
 
   @Test
+  public void customExpanderListParam() throws Exception {
+    server.enqueue(new MockResponse());
+
+    TestInterface api = new TestInterfaceBuilder().target("http://localhost:" + server.getPort());
+
+    api.expandList(Arrays.asList(new Date(1234l)));
```

Comment:
if folks are interested in this change, I'd test multiple values and also skipping of null. that way, the key behaviors are locked and understood (even if debatable)